### DISCUSSION TITLE
[Feature]Make AttachedCardShadow available for 1809 update

### DISCUSSION THREAD

**Discussion by AbdAlghaniAlbiek** [CommunityToolkit/Labs-Windows#243] [Ideas]
## Describe the problem this feature would solve
Hi @Ryken100, sorry to bother you but I was searching in the issues and pull requests and I didn't found  pull request for updating AttachedCardShadow to accomplish 1809 update, you told me that the reason of working of this type of shadow only on May 2019 update and above that it's depending on CompositionVisualSurface and there is way to make it working on October 2018 update and there is will be a future update to accomplish this

## Describe the solution
Making  AttachedCardShadow working on 1809 update

**Comment by ghost** [comment]
Hello, 'AbdAlghaniAlbiek! Thanks for submitting a new feature request. I've automatically added a vote 👍 reaction to help get things started. Other community members can vote to help us prioritize this feature in the future!

**Comment by Ryken100** [comment]
Hi @AbdAlghaniAlbiek, that will be the next thing I will work on :)

**Comment by michael-hawker** [comment]
@Ryken100 is CommunityToolkit/WindowsCommunityToolkit#4404 what would address this then or is the CompositionVisualSurface still used? We need a tracking issue for the new PR, so just wondering if we could re-use this one or not.

**Comment by Ryken100** [comment]
@michael-hawker that PR doesn't address this one

**Comment by AbdAlghaniAlbiek** [comment]
@Ryken100 I'm sorry if you are mad from me if I made you forced to do this improvement
do you want to close this issue ??

**Comment by michael-hawker** [comment]
@AbdAlghaniAlbiek I don't think @Ryken100 is mad? I was just clarifying the PR and issue alignment for tracking purposes? I think it was his intent to potentially look at this anyway and you just happened to file an issue at the same time. 🙂
